crates/typst/src/layout/layout.rs

```diff
@@ -56,8 +59,8 @@ pub fn layout(
     /// and `height`.
     ///
     /// This function is called once for each time the content returned by
-    /// `layout` appears in the document. That makes it possible to generate
-    /// content that depends on the size of the container it is inside of.
+    /// `layout` appears in the document. This makes it possible to generate
+    /// content that depends on the size of its container.
```

Comment:
```suggestion
    /// content that depends on the dimensions of its container.
```
The other mentions of "size" have also been replaced, so this one should probably too.